packages/dapp/src/components/Main.tsx

```diff
@@ -31,7 +31,7 @@ class Main extends React.Component<DispatchProp<any> & RouteComponentProps<any>>
 
   public onNetworkUpdated = async (): Promise<void> => {
     const civil = getCivil();
-    if (civil.network) {
+    if (civil.network === "4") {
       this.props.dispatch!(setNetwork(civil.network));
```

Comment:
I think we need to set the network even if it's not rinkeby so that we properly render the error ("switch to rinkeby") bar